client/src/components/dashboard/PositionCard.js

```diff
@@ -31,9 +32,20 @@ const PositionCard = (props) => {
   const classes = useStyles();
   const { position } = props;
   const { name, imageLink } = position;
+  const [open, setOpen] = useState(false);
+
+  const handleClickOpen = () => {
+      setOpen(true);
+  }
+
+  const handleClose = () => {
+      setOpen(false);
+  }
+
   return (
     <Paper className={classes.root}>
```

Comment:
If you add `onClick={handleClickOpen}` to the paper component it should activate the notation display if they've clicked on the card. 
```suggestion
    <Paper className={classes.root} onClick={handleClickOpen}>
```

Reply:
Looks like it might be because it doesn't work on the parent component, so I encased the card into a Box component and added the onClick function to the Box - the entire tile should be clickable and open up the dialog now